functions/backup.bash

```diff
@@ -474,6 +474,8 @@ setup_mirror_SD() {
   local serviceTargetDir="/etc/systemd/system"
   local sizeError="your destination SD card device does not have enough space"
   local storageDir="${storagedir:-/storage}"
+  local storageRemovalQuery="Do you also want to remove the storage mount for ${storageDir} ?\\nIf you do not but remove the physical device it is located on, you will have trouble every time you restart your system.\\nRemember though it might also contain data you might want to keep such as your Amanda backup data. If ${storageDir} is not where your mount is, stop now and enter your mountpoint in /etc/openhabian.conf as the storagedir= parameter ."
```

Comment:
```suggestion
  local storageRemovalQuery="Do you also want to remove the storage mount for ${storageDir}?\\nIf you do not but remove the physical device it is located on, you will have trouble every time you restart your system.\\nRemember though it might also contain data you might want to keep such as your Amanda backup data. If ${storageDir} is not where your mount is, stop now and enter your mountpoint in /etc/openhabian.conf as the `storagedir=parameter`."
```